lib/ansible/modules/windows/win_nssm.py

```diff
@@ -87,17 +87,19 @@
       - Password to be used for service startup
   start_mode:
     description:
-      - If C(auto) is selected, the service will start at bootup. C(manual) means that the service will start only when another service needs it.
+      - If C(auto) is selected, the service will start at bootup. C(delayed) causes a delayed but automatic start after boot. C(manual) means that the service will start only when another service needs it.
```

Comment:
After every `.`, put it on a newline and start with `-`, e.g.

```
description:
- If C(auto) is selected, ...
- If C(delayed) ...
- If C(manual)...
```
I would also put in a new entry saying `delayed` was added in 2.5.